src/NServiceBus.Core/Routing/MessagingBestPractices/BestPracticesFeature.cs

```diff
@@ -0,0 +1,47 @@
+namespace NServiceBus.Features
```

Comment:
@andreasohlund do you remember the reason we decided to remove the best practice feature and merge it with routing? I know we discussed that not a long time ago